ppr-api/src/ppr_api/resources/party_codes.py

```diff
@@ -13,22 +13,23 @@
 # limitations under the License.
 """API endpoints for maintaining preset client registering and secured parties."""
 
+# pylint: disable=too-many-return-statements
+
 from http import HTTPStatus
 
-from flask import abort, g, request
+from flask import request
 from flask_restplus import Namespace, Resource, cors
```

Comment:
i must have missed this earlier, should be restx, or just plain flask

Reply:
Okay, I can make the change later - I used the dependencies from lear.